apps/confidential/confidential_test.go

```diff
@@ -138,6 +138,7 @@ func TestAcquireTokenByCredential(t *testing.T) {
 		}
 		client, err := fakeClient(accesstokens.TokenResponse{
 			AccessToken:   token,
+			RefreshOn:     internalTime.DurationTime{T: time.Now().Add(1 * time.Hour)},
```

Comment:
This is not a realistic refresh_in value. eSTS will set this at 1/2 expires_in value.

Reply:
Comment not resolved.